modules/configuring/proc-adding-custom-configuration-files-by-using-the-operator.adoc

```diff
@@ -0,0 +1,48 @@
+[id="proc-add-custom-app-config-file-ocp-operator_{context}"]
+= Adding your custom {product} configuration to a running {product-short} by using the operator
+
+Reference your custom config maps and secrets in your the `Backstage` custom resource
```

Comment:
"Reference your custom config maps and secret in the `Backstage` custom resource"
or
"Reference your custom config maps and secret in your `Backstage` custom resource"

Reply:
```suggestion
Reference your custom config maps and secrets in your `Backstage` custom resource
```